libpod/adapter/runtime_remote.go

```diff
@@ -320,6 +322,129 @@ func (r *LocalRuntime) Config(name string) *libpod.ContainerConfig {
 	return &data
 
 }
+
+// PruneImages is the wrapper call for a remote-client to prune images
 func (r *LocalRuntime) PruneImages(all bool) ([]string, error) {
 	return iopodman.ImagesPrune().Call(r.Conn, all)
 }
+
+// Export is a wrapper to container export to a tarfile
+func (r *LocalRuntime) Export(name string, path_ string) error {
```

Comment:
This isn't C - where's the _ on the end of `path_` coming from?

Reply:
resolved.